config/locales/ar.yml

```diff
@@ -3626,10 +3626,6 @@ ar:
     rma_credit: "رصيد RMA"
     refund: "إعادة المال"
     server_error: "خطأ في الخادم"
-    shipping_method_names:
-      UPS Ground: "UPS الأرضي"
-      pick_up: "التسليم من المزرعة مباشرة"
-      delivery: "موقعة ومختومة وتم التسليم"
```

Comment:
You didn't need to change all the locales. That change will be applied automatically when we pull the translation files from Transifex when preparing a release.